### PR TITLE
Add wagtail localize

### PR DESCRIPTION
**Pull request by Pomax**
Builds on https://github.com/mozilla/foundation.mozilla.org/pull/6085 to add wagtail-localize, after that PR removes wagtail-modeltranslation

Note that this only adds wagtail-localize, and not the additional git/pontoon functionality.

### CODE REVIEW

**Comment by Pomax** [comment]
It looks like the UI is currently still the "all pages in all locales" UI, so we'll need to update this to the new version of wagtail-localize when that gets pushed up.

image

**Comment by Pomax** [comment]
It also looks like I can still edit fields that are marked as synchronized? For example, the blox index page `slug` value:

image

**Comment by TheoChevalier** [comment]
> It also looks like I can still edit fields that are marked as synchronized? For example, the blox index page `slug` value is marked as "uses English version" but still has an edit button. (this might be by design?)
> 
> image

Right, it can still be overridden, which is still an improvement to the current situation where it can be changed without the user noticing :)

I think we should keep it like that for now, and later use the “Ignore field” option that Karl is going to add for us, otherwise we will have long `translatable_fields` lists in most classes

**Comment by TheoChevalier** [comment]
> It looks like the UI is currently still the "all pages in all locales" UI, so we'll need to update this to the new version of wagtail-localize when that gets pushed up.
> 
> image

The language switcher is/will be part of Wagtail core, so it will actually require a Wagtail upgrade (2.13?) rather than a wagtail-localize one

**Review by TheoChevalier — APPROVED** [review]
LGTM, just noting we can remove the initial attempt to fix PNI

**Review by KalobTaulien — APPROVED** [review]

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 1]
```diff
@@ -361,6 +363,11 @@ class ProductPage(AirtableMixin, FoundationMetadataPageMixin, Page):
         related_name='votes',
     )
 
+    override_translatable_fields = [
+        SynchronizedField('votes'),
+        SynchronizedField('creepiness_value'),
+    ]
+
```
[the remark this answers is not in the document]
Comment (Pomax) [reply network-api/networkapi/wagtailpages/pagemodels/products.py:370]:
```suggestion
```

**Pull request merged**